lib/gssapi/krb5/export_sec_context.c

```diff
@@ -247,7 +247,7 @@ _gsskrb5_export_sec_context(
     ret = _gsskrb5_delete_sec_context (minor_status, context_handle,
 				       GSS_C_NO_BUFFER);
     if (ret != GSS_S_COMPLETE)
-	_gss_secure_release_buffer (&minor, interprocess_token);
+	_gsskrb5_release_buffer (minor_status, interprocess_token);
```

Comment:
This change should not be included as it clobbers the possibly-useful value of `*minor_status`.

Reply:
Also, you want to zero it.